rs/crypto/BUILD.bazel

```diff
@@ -39,7 +39,7 @@ DEPENDENCIES = [
     "//rs/types/base_types",
     "//rs/types/types",
     "@crate_index//:bincode",
-    "@crate_index//:clap_3_2_25",
+    "@crate_index//:clap",
```

Comment:
This line actually seems obsolete because it is only used in `rs/crypto/src/bin/ic-crypto-csp.rs` which has its own `rust_binary` entry with its own dependency on clap. I tried to compile without it, and it worked.

Reply:
Done thanks